GVRf/Sample/scene-objects/src/org/gearvrf/sample/sceneobjects/SceneObjectActivity.java

```diff
@@ -89,13 +114,47 @@ private void createCameraView() {
             camera = Camera.open();
             if (camera != null) {
                 Parameters params = camera.getParameters();
-                params.setFocusMode(Parameters.FOCUS_MODE_CONTINUOUS_VIDEO);
-                camera.setParameters(params);
-                camera.startPreview();
-            }
-        } catch (Exception exception) {
-            android.util.Log.d(TAG, "Camera not available or is in use");
-        }
+                
```

Comment:
can we add a method to GVRCameraSceneObject to set all these parameters to make it easier for developers to set this up?